http-client.ts

```diff
@@ -175,6 +171,30 @@ export class HttpClient implements Server.IHttpClient {
 		}).future<Server.IResponse>()();
 	}
 
+	private setResponseResult(result: IFuture<Server.IResponse>, timerId: number, resultData: { response?: Server.IRequestResponseData, body?: string, err?: Error }): void {
+		if (timerId) {
+			clearTimeout(timerId);
+			timerId = null;
```

Comment:
In fact it's better to keep it, the timerId is in fact NodeJS.Timer, but the TypeScript thinks it's number (as the result of setTimeout is set to number due to TypeScript definitions)